workers/approov-token-binding/README.md

```diff
@@ -0,0 +1,174 @@
+
+
+We need an [Approov secret](https://approov.io/docs/latest/approov-cli-tool-reference/#secret-command) to check the signature in the JWT tokens and we need to use the same one used by the [Approov Cloud service](https://www.approov.io/approov-in-detail.html) to sign the [Approov Tokens](https://www.approov.io/docs/latest/approov-usage-documentation/#approov-tokens) issued to our mobile app.
+
+#### Retrieve the Approov Secret
+
+```text
+approov secret /path/to/approov/administration.token -get base64
+```
+
+> **NOTE:** The Approov secret requires a management admin token, because the developer management token doesn't have permissions to get the secret.
+
+#### Set the Approov Base64 Secret in Cloudflare
+
+From the root of `workers/approov-token-binding/` folder run:
+
+```text
+wrangler secret put --env staging APPROOV_BASE64_SECRET
+```
+
+> **NOTE:**: Please bear in mind that you need to create the environment variable `APPROOV_BASE64_SECRET` for each environment you want to run the Approov token binding worker.
+
+#### Set the Approov Token Binding Header Name in Cloudflare
+
+The default value for `TOKEN_BINDING_HEADER_NAME` in the code is `Authorization`, therefore you just need to set it in CLoudflare as an environment variable if you plan to use a different header name.
+
+To use a different header name run from the root of `workers/approov-token-binding/` folder:
+
+```text
+wrangler secret put --env staging TOKEN_BINDING_HEADER_NAME
+```
+
+> **NOTE:**: Please bear in mind that you need to create the environment variable `TOKEN_BINDING_HEADER_NAME` for each environment you want to run the Approov token binding worker.
+
+
+#### View the Secrets in the Cloudfare Dashboard
+
+Now you can visit the Cloudflare Dashboard, select the tab **Workers**, click in the worker name, and then select the tab **Settings** to see the `APPROOV_BASE64_SECRET` and `TOKEN_BINDING_HEADER_NAME` listed as environment variables.
+
+
+### Test it Works
+
+The following examples use cURL, but you can use instead the [Postman Collection](/README.md#api-requests-with-postman) to make the API requests, just remember that you need to adjust the urls and replace the tokens in the collection or instead you can use the dummy secret used to sign them, just follow the instructions.
+
+
+#### With a Valid Approov Token
+
+Generate a valid token example from the Approov Cloud service:
+
+```
+approov token -setDataHashInToken 'Bearer theuthorizationtokenhere' -genExample your.api.domain.com
+```
+
+Then make the request with the generated token:
+
+```
+curl -i --request GET 'https://your.api.domain.com/v1/shapes' \
+  --header 'Authorization: Bearer theuthorizationtokenhere' \
+  --header 'Approov-Token: APPROOV_TOKEN_EXAMPLE_HERE'
+```
+
+The request must be ok:
+
+```text
+HTTP/2 200
+
+...
+
+{
+  "form": "Sphere"
+}
+```
+
+#### With an Invalid Approov Token
+
+Let's just remove the Authorization request from the request:
```

Comment:
Wouldn't a better example be a different authz token?

Reply:
I will add it to.